doc/overview.rst

```diff
@@ -0,0 +1,40 @@
+.. _overview:
+.. include:: ./ext_links.txt
+
+Overview
+========
+
+.. module:: dpnp
+
+The Data Parallel Extension for NumPy* (dpnp package) - a library that
+implements a subset of `NumPy*`_ that can be executed on any
+data parallel device. The subset is a drop-in replacement of core `NumPy*`_
+functions and numerical data types.
+
+The Data Parallel Extension for NumPy* is being developed as part of
+`Intel AI Analytics Toolkit`_ and is distributed with the
+`Intel Distribution for Python*`_. The dpnp package is also available
+on Anaconda cloud. Please refer the :doc:`getting_started` page to learn more.
```

Comment:
The reference to this page is not displayed correctly
<img width="350" alt="image" src="https://github.com/IntelPython/dpnp/assets/68376232/9996cb64-8f20-4d1f-a895-d2c01b4af6b1">

Reply:
Done